scripts/mws-integration/main.tf

```diff
@@ -55,6 +55,53 @@ module "aws_common" {
   tags = local.tags
 }
 
+resource "aws_s3_bucket" "logdelivery" {
```

Comment:
nit: can we replace logDelivery with log_delivery?

Reply:
it will force deletion and re-creation of all relevant resources, which i'm rather avoiding